pkg/nodejs/npm/parse.go

```diff
@@ -71,10 +78,11 @@ func (p *Parser) parse(dependencies map[string]Dependency, dircetDeps map[string
 		}
 
 		lib := types.Library{
-			ID:       p.ID(pkgName, dependency.Version),
-			Name:     pkgName,
-			Version:  dependency.Version,
-			Indirect: isIndirectLib(pkgName, dircetDeps),
+			ID:                 p.ID(pkgName, dependency.Version),
+			Name:               pkgName,
+			Version:            dependency.Version,
+			Indirect:           isIndirectLib(pkgName, dircetDeps),
+			ExternalReferences: p.GetExternalRefs(dependency),
```

Comment:
We can directly put a reference here.
```suggestion
			ExternalReferences: []types.ExternalRef{
				{Type: types.Other, Url: dependency.Resolved},
			}
```